Made Services Page

Abdullah-Waqas13341 opened the pull request:
Made Form to Post Services
Adjusted Routes

Commit:
Made Services Page
Made Form to Post Services
Adjusted Routes

rabbiasajjad-devsinc commented on sellers/views.py:
Add error handling using try-except in case the form validation fails

rabbiasajjad-devsinc replied:
You can put form validation in a separate method as well.

Abdullah-Waqas13341 replied:
ok

rabbiasajjad-devsinc commented on sellers/views.py:
Follow PEP import guidelines for the best practices

https://peps.python.org/pep-0008/#:~:text=Imports,-Imports%20should%20usually&text=Imports%20are%20always%20put%20at,Standard%20library%20imports.

Commit:
Added the following Functionality
View Posted Serivces
View Booked Services
Payment Stays Check
Alterd Booking Model to add Payment Status Attribute
Functionality to Apporove or Reject Posted Services of Seller by Admin

Commit:
Added Features
Adding Functionality to Track Progress

Commit:
Altered login form to just keep Email and Password and Removed Username Field

Commit:
Added Features
Added Custom Admin View
Admin View now only contains Services ,Category and Notifcation Option (To Show Pending Services)
To get Access to Super Admin go to (http://127.0.0.1:8000/superadmin/) and add username and password

Commit:
Added Feature
Added a Button that will send a request for reapproval to the admin
Temporarly removed the Functionality of Super Admin Access from the Previous Commit

Commit:
Corrected Error
Fixed User Locout After Unsuccessful Login Attempts
Show Reviews to the Sellers is still remaining

Commit:
Final Commit
